git/git.go

```diff
@@ -573,6 +573,32 @@ func GetCommitSummary(commit string) (*CommitSummary, error) {
 	}
 }
 
+func isCygwin() bool {
+	cmd := subprocess.ExecCommand("uname")
+	buf := &bytes.Buffer{}
+	cmd.Stderr = buf
+	out, err := cmd.Output()
+	output := string(out)
+	if (err == nil) && (strings.Contains(output, "CYGWIN")) {
+		return true
+	} else {
+		return false
+	}
+}
+
+func translateCygwinPath(path string) (string, error) {
+	cmd := subprocess.ExecCommand("cygpath", "-w", path)
+	buf := &bytes.Buffer{}
+	cmd.Stderr = buf
+	out, err := cmd.Output()
+	output := string(out)
+	output = strings.TrimSpace(output)
+	if err != nil {
+		return path, fmt.Errorf("Failed to translate path from cygwin to windows")
```

Comment:
This should show the stderr contents:

    return path, fmt.Errorf("Failed to translate path from cygwin to windows: %s", buf.String())